paasta_tools/cleanup_kubernetes_jobs.py

```diff
@@ -124,7 +124,9 @@ def instance_is_not_bouncing(
         if isinstance(application, DeploymentWrapper):
             existing_app = application.item
             if (
-                existing_app.metadata.namespace == instance_config.get_namespace()
+                existing_app.metadata.namespace != instance_config.get_namespace()
+                and (instance_config.get_bounce_method() == "downthenup")
+                or existing_app.metadata.namespace == instance_config.get_namespace()
```

Comment:
I can never remember the precedence of `and`/`or` - can you add some parentheses here?